### ISSUE TITLE
Render window exaggeration

### ISSUE THREAD

**Issue by banesullivan** [akaszynski/vtki#39]
Is it possible to exaggerate an axis in a VTK rendering window? If so, could we implement a method to do so?

I ask because this is very common (almost necessary) for many geoscientific applications. Often we're dealing with 10s-100s of square *kilometers* in the lateral (XY-plane) while looking only in the 10s-100s of *meters* in the Z-direction. These proportions make visualization rather difficult and exaggeration would help significantly

**Comment by akaszynski** [comment]
I'd just modify the points in-place with:
```python

# scale Z dimension by 1000
import vtki
sphere = vtki.Sphere()
sphere.points[:, 2] *= 1000
```

You'd have to keep the original points stored somewhere.  The other way is to keep the points unchanged and to the ``SetScale`` method of the actor.  Seems cleaner now that I think about it.  See:
[vtkProp3D SetScale](https://vtk.org/doc/nightly/html/classvtkProp3D.html#a6052d85c7ef109800a859c56959650d3).

You could use a widget to implement a slider bar that calls back axis scaling too.

**Comment by banesullivan** [comment]
Oh! Easy! This fits perfectly into #43 as I added a place to keep a pointer to all the actors in the scene.

One final issue I'm running into: how do we appropriately scale the `vtkCubeAxes`? I made a new function:

```py
def set_scale(self, xscale=1.0, yscale=1.0, zscale=1.0):
        """Scale all the datasets in the scene. 
        Scaling in performed independently on the X, Y and Z axis.
        A scale of zero is illegal and will be replaced with one.
        """
        for actor in self._actors:
            actor.SetScale(xscale, yscale, zscale)
        self.update_bounds_axes()
        self._render()
```

But when the cube axes gets scaled, the data range is increased by the scaling factor

**Comment by banesullivan** [comment]
**Update:** The axis can now be scaled with `.set_scale(x, y, z)` and I'll be closing this issue for Milestone v0.15.0

There is however a bug that appears to be on the VTK backend for ` vtkCubeAxesActor.SetScale` which prevents the axes bound from displaying scaled coordinates. This might confuse users and should be kept in mind.

```py
p = vtki.BackgroundPlotter()

p.add_mesh(examples.load_hexbeam())
p.add_bounds_axes()
```

Without a scaling:

![no-scale](https://user-images.githubusercontent.com/22067021/51217667-2db6a500-18e6-11e9-94d8-15b6e5c1efe1.png)

After a scaling:

```py
p.set_scale(4,4,1)
```

![scale](https://user-images.githubusercontent.com/22067021/51217666-2db6a500-18e6-11e9-9ad1-9d3b0873ac85.png)

**Comment by banesullivan** [comment]
Fixed in #86
